openroad_interface/place_n_route.py

```diff
@@ -0,0 +1,48 @@
+import os 
+
+import networkx as nx
+
+from var import directory
+import length_calculations as lc
+import parasitic_calc as pc
+
+def place_n_route(graph, design_name: str, net_out_dict: dict, node_output: dict, lef_data: dict) -> dict:
```

Comment:
I want openroad to be called within this function. 
And maybe renambe this function to detailed place_n_route